src/node_ffi.h

```diff
@@ -0,0 +1,80 @@
+#ifndef SRC_NODE_FFI_H_
+#define SRC_NODE_FFI_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+#include "env.h"
+#include "ffi.h"
+
+namespace node::ffi {
+using binding::DLib;
+using permission::PermissionScope;
+using v8::Array;
+using v8::ArrayBuffer;
+using v8::BackingStore;
+using v8::BigInt;
+using v8::Boolean;
+using v8::Context;
+using v8::External;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::Int32;
+using v8::Isolate;
+using v8::Local;
+using v8::Number;
+using v8::Object;
+using v8::Persistent;
+using v8::Uint32;
+using v8::Value;
```

Comment:
In header files we use the fully qualified names. This list of using decls needs to be moved into the c++ file